level2/autoscaling.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_autoscaling_group" "app-asg" {
+  launch_configuration = aws_launch_configuration.web_config.id
+  vpc_zone_identifier  = data.terraform_remote_state.networking.outputs.public-subnet_id
+
+  load_balancers    = ["${aws_elb.app-lb.name}"]
+  health_check_type = "ELB"
```

Comment:
```suggestion
```

Reply:
Should i remove it at all??

Reply:
Yes, it's an optional field. By default it is EC2 which is a way to go

Reply:
Alright